### ISSUE TITLE
Get error "Uncaught Invariant Violation: .........."!

### ISSUE THREAD

**Issue by thearabbit** [coniel/meteor-react-form-handler#13]
I use it with "Flow Router (SSR)".
Get error
```
Uncaught Invariant Violation: addComponentAsRefTo(...): Only a ReactOwner can have refs. You might be adding a ref to a component that was not created inside a component's `render` method, or you have multiple copies of React loaded (details: 
```

**Comment by coniel** [comment]
Hi, I'm on holiday at the moment. Won't be able to look at this until Wednesday.

**Comment by thearabbit** [comment]
Thanks for your reply.
Waiting ...

**Comment by hunius** [comment]
same here...

**Comment by coniel** [comment]
Sorry, been a bit overwhelmed lately and haven't had time to address issues.

I don't use FlowRouter myself, so it's hard to say whether that is the problem. I think it might be because there are multiple react versions running (I had that issue which was easy to solve with webpack). How are you guys adding react to your project, and which form UI package  are you using (semantic or material ui)?

**Comment by hunius** [comment]
Hey, no problem.
I am using Mantra and have added react (v 0.14.7) and material-ui (v 0.14.4) by npm. The UI Package I am using is material. Don't know if this is a FlowRouter issue. 
Here is my npm ls | grep 'react':

├── babel-plugin-react-require@2.1.0 grep 'react'
├─┬ babel-preset-react@6.5.0
│ ├─┬ babel-plugin-transform-react-display-name@6.5.0
│ ├─┬ babel-plugin-transform-react-jsx@6.6.5
│ │ ├─┬ babel-helper-builder-react-jsx@6.6.5
│ └─┬ babel-plugin-transform-react-jsx-source@6.5.0
├── eslint-plugin-react@3.15.0
│ ├─┬ react-komposer@1.3.2
│ │ ├── hoist-non-react-statics@1.0.5
│ └─┬ react-simple-di@1.0.1
│   └── react-display-name@0.0.1
├─┬ react@0.14.7
├── react-addons-test-utils@0.14.7
├─┬ react-dnd@2.1.2
├── react-dnd-html5-backend@2.1.2
├── react-dom@0.14.7
├─┬ react-dropzone-component@0.8.1
├── react-mounter@1.1.0
├─┬ react-tap-event-plugin@0.2.2
├─┬ react-tree-menu@1.5.0


Looks ok to me. Maybe you have an idea?
Thanks a lot for your work!

**Comment by coniel** [comment]
Yep, I'm pretty sure it's because you are loading react from npm and this package uses Meteor's react package, so it's causing two version of react to be loaded.

I'll publish this package to npm so that you can add it from there instead, which should take care of the issue. Will do it within a couple of hours, I'll post back when it's up.
